analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -272,24 +273,26 @@ def main(args):
                                            profile_checkers)
 
         for checker_name, value in config_handler.checks().items():
-            enabled, description = value
+            state, description = value
 
-            if not enabled and 'profile' in args:
+            if state != CheckerState.enabled and 'profile' in args:
                 continue
 
-            if enabled and 'only_disabled' in args:
+            if state == CheckerState.enabled and 'only_disabled' in args:
                 continue
-            elif not enabled and 'only_enabled' in args:
+            elif not state != CheckerState.enabled and 'only_enabled' in args:
                 continue
 
             if args.output_format != 'json':
-                enabled = '+' if enabled else '-'
+                state = '+' if state == CheckerState.enabled else '-'
```

Comment:
Can be the `state` equal with `CheckerState.default` here? In this case it will be marked with `'-'`.

Reply:
This is intended. I have added comments to CheckerState to clarify the 3 states of enabled, default, disabled. In the metadata, the following mapping is used: CheckerState.enabled -> enabled (True), other cases map to False.